src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/CreateCaseTest.java

```diff
@@ -0,0 +1,204 @@
+import uk.gov.hmcts.reform.bulkscan.orchestrator.dm.DocumentManagementUploadService;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.helper.CaseSearcher;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.helper.CcdCaseCreator;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.helper.EnvelopeMessager;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.CcdApi;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.CcdAuthenticator;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.CcdAuthenticatorFactory;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.events.CreateExceptionRecord;
+import uk.gov.hmcts.reform.ccd.client.model.AboutToStartOrSubmitCallbackResponse;
+import uk.gov.hmcts.reform.ccd.client.model.CallbackRequest;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.logging.appinsights.SyntheticHeaders;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.UUID;
+import java.util.concurrent.TimeUnit;
+
+import static org.apache.http.HttpHeaders.AUTHORIZATION;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.awaitility.Awaitility.await;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@SpringBootTest
+@ActiveProfiles("nosb") // no servicebus queue handler registration
+@Disabled
+class CreateCaseTest {
+
+    @Value("${test-url}")
+    private String testUrl;
+
+    @Value("${document_management.url}")
+    private String documentManagementUrl;
+
+    @Value("${document_management.context-path}")
+    private String dmContextPath;
+
+    @Autowired
+    private CcdApi ccdApi;
+
+    @Autowired
+    private CcdCaseCreator ccdCaseCreator;
+
+    @Autowired
+    private CaseSearcher caseSearcher;
+
+    @Autowired
+    private EnvelopeMessager envelopeMessager;
+
+    @Autowired
+    private DocumentManagementUploadService dmUploadService;
+
+    @Autowired
+    private CcdAuthenticatorFactory ccdAuthenticatorFactory;
+
+    private String dmUrl;
+
+    @BeforeEach
+    void setUp() throws Exception {
+
+        dmUrl = dmUploadService.uploadToDmStore(
+            "Certificate.pdf",
+            "documents/supplementary-evidence.pdf"
+        );
+    }
+
+    @Test
+    public void should_create_case_from_valid_exception_record() throws Exception {
+        // given
+        CaseDetails exceptionRecord = createExceptionRecord("envelopes/exception-with-evidence.json");
+
+        // when
+        String caseCcdId = invokeCallbackEndpoint(exceptionRecord, "ccdCaseReference");
+
+        // then
+        await("Case is created")
+            .atMost(60, TimeUnit.SECONDS)
+            .pollDelay(2, TimeUnit.SECONDS)
+            .until(() -> isCaseCreated(caseCcdId, exceptionRecord));
+
+        verifyCaseIsCreated(caseCcdId, exceptionRecord);
```

Comment:
In its current form we probably don't need this method, as it only checks if the case has been created - something that we already know at this point. But maybe it would be good for verification of some of the case's details?

Reply:
I will leave it here until we can run the test

Reply:
Maybe we should remove this call, then. The previous statement will throw an exception if the case isn't created.

Worth adding a TODO comment to verify case details.

Reply:
Have done